pkg/config/config.go

```diff
@@ -904,6 +905,7 @@ func InitConfig(config Config) {
 
 	setAssetFs(config)
 	setupAPM(config)
+	appsecconfig.InitConfig(config)
```

Comment:
I think that for the sake of consistency, all of the "set known" and "bind env" stuff should be in this package, as it is for all other agents. Perhaps you can have a `setupAppsec` function in a separate file which can have your team as the owner.

Reply:
Fixed: move `pkg/appsec/config.InitConfig()` to `pkg/config.setupAppSec()` in a new `appsec.go` file (added to CODEOWNERS too)